src/types/onyx/Report.ts

```diff
@@ -71,6 +71,9 @@ type Report = OnyxCommon.OnyxValueWithOfflineFeedback<
         /** The URL of the Group Chat report custom avatar */
         avatarUrl?: string;
 
+        /** The file name of the Group Chat report custom avatar */
+        originalFileName?: string;
```

Comment:
This is not returned by the BE. Let's not add it. Also this does not explain why the view avatar option works online

Reply:
Make sense. removing this. but we was not seeing avatar offline because we don't have fileName in report.

Reply:
I checked and this seems to be related to filename. As we don't have `originalFileName` in report onyx object [here](https://github.com/Expensify/App/blob/530df19f6e025806c8888848b3cc0e8d23b65ee1/src/pages/ReportAvatar.tsx#L46) so it does not pass the condition here:
https://github.com/Expensify/App/blob/530df19f6e025806c8888848b3cc0e8d23b65ee1/src/components/Attachments/AttachmentView/index.tsx#L199-L200
and that's why it is showing falling back to default avatar component.

The logs for online and offline:

| Online  | Offline |
| ------------- | ------------- |
| <img width="1512" alt="Screenshot 2024-06-24 at 2 09 43 PM" src="https://github.com/Expensify/App/assets/53095363/5fe283e6-05b1-4b32-a7d2-c3f72843faef"> | <img width="1512" alt="Screenshot 2024-06-24 at 2 08 37 PM" src="https://github.com/Expensify/App/assets/53095363/f5d04c9b-1abb-4e66-ae24-cced6e4e851e">|

Note: that the difference in filename of offline and online is `.jpeg` and that why it is passing the isImage( ) condition.

Reply:
Ah I see it now. Can you please rename the field to `avatarFileName` and add a comment stating that this is field is local only

Reply:
Also please merge main and fix the conflicts (note that now we have `avatarFileName` in `NewGroupChatDraft`)